src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,48 @@
-export const Tabs = () => {};
+import { Tab } from '../../types/Tab';
+
+type Props = {
+  tabs: Tab[],
+  selectedTabId: string,
+  onTabSelected: (newId: string) => void,
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const selectedTab = tabs.find(tab => tab.id === selectedTabId);
+
+  if (!selectedTab) {
```

Comment:
It's not advised to call the onTabSelected prop directly in the render function, as this might cause unwanted side effects. Consider using useEffect to update the selected tab when the component mounts or the selectedTabId prop changes.